bot/cogs/fun.py

```diff
@@ -199,6 +199,47 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @command(aliases=["leet"])
+    async def leetify(self, ctx: Context, *, content: str) -> None:
+        """Gives each letter of a given message a different markdown style."""
+
+        leetters = {
+            "a": ["A","4"], "b": ["b", "B"], "c": ["C", "c"], "d": ["D", "d"],
+            "e": ["E", "3"], "f": ["F", "f"], "g": ["G", "g"], "h": ["H", "h"],
+            "i": ["1", "I", "i"], "j": ["j", "J"], "k": ["K", "k"], "l": ["L", "l"],
+            "m": ["M", "m"], "n": ["n", "N"], "o": ["O", "0"], "p": ["P", "p"],
+            "q": ["Q", "q"], "r": ["R", "r"], "s": ["5", "S"], "t": ["t", "T"],
+            "u": ["U", "u"], "v": ["v","V"], "w": ["W", "w"], "x": ["X", "x"],
+            "y": ["y", "Y"], "z": ["Z", "z"]
+        }
```

Comment:
As I've pointed out in a review before, I'm personally not a fan of this dictionary since in most of the cases it simply follows this: `letter: [letter, letter.upper()]`, we don't need to repeat ourselves here and generate a dictionary for every single letter in the alphabet, we can make it easier and only define the special cases like `"a": ["A", "4"]` and for those which won't be there simply default it to a random choice of that same letter or uppercase version of it.
It would look something like this:
```py
leetified_word = ""
# In python you can loop through individual letters by doing `for letter in string`
for letter in content:
    # The letter is in the dictionary (that means it is one of the special cases where we don't only use random choice of lowercase and uppercase version of that letter)
    if letter in letters:
       leetified_word += random.choice(leeters[letter])
    # If the letter isn't defined in the leeters, use the upper or lowercase version randomly
    else:
        leetified_word += random.choice([letter, letter.upper()])
```
This suggestion is basically a much simpler version of most of the proposed function

Reply:
thanks to your review, i've shortened the dictionary length to 5 items.